avocado/utils/multipath.py

```diff
@@ -115,6 +115,48 @@ def get_paths(wwid):
     return paths
 
 
+def get_multipath_details():
+    """
+    Get multipath details as a dictionary, as given by the command:
+    multipathd show maps json
+
+    :return: Dictionary of multipath output in json format.
+    """
+    mpath_op = process.system_output("multipathd show maps json", shell=True)
+    mpath_op = ast.literal_eval(mpath_op.replace("\n", '').replace(' ', ''))
+    return mpath_op
+
+
+def is_path_a_multipath(disk_path):
+    """
+    Check if given disk path is part of a multipath.
+
+    :param disk_path: disk path. Example: sda, sdb.
+
+    :return: True if part of multipath, else False.
+    """
+    if not process.system("multipath -c /dev/%s" % disk_path, shell=True,
+                          ignore_status=True):
+        return True
+    return False
+
+
+def get_path_status(disk_path):
```

Comment:
Sure, makes sense. Will take care of this once `get_multipath_details()` is ready.